src/coreclr/src/nativeresources/resourcestring.cpp

```diff
@@ -37,7 +37,7 @@ int LoadNativeStringResource(const NativeStringResourceTable &nativeStringResour
 
         if (resourceEntry != NULL)
         {
-            len = PAL_GetResourceString(NULL, resourceEntry->resourceString, szBuffer, iMax);
+            len = UTF8ToUnicode(resourceEntry->resourceString, strlen(resourceEntry->resourceString) + 1, szBuffer, iMax, 0);
```

Comment:
```suggestion
            len = MultiByteToWideChar(CP_UTF8, 0, resourceEntry->resourceString, -1, szBuffer, iMax);
```
Can you please use this so that we don't need to export new function from PAL?